middlewares/auth-verify.middleware.js

```diff
@@ -0,0 +1,16 @@
+const jwt = require("jsonwebtoken");
+const jwtSecret = process.env['jwt-secret'];
```

Comment:
Just a suggestion, it's a convention to have env variable names in capital case.